_includes/functions/details-list.md

```diff
@@ -0,0 +1,25 @@
+{% capture /dev/null %}
+  <!-- Maximum number of params:
+     1 include.class (CSS class)
+    10 include.q0..q9 (questions)
+    10 include.a0..a9 (answers)
+    -- -------------------
+    21 Total -->
+  {% assign includes_size = include | size %}
+  {% if includes_size >= 21 %}{% include ERROR143_too_many_params_passed_to_details_list %}{% endif %}
+{% endcapture %}
+<div class="{{include.class | default: 'qa_details'}} {{jekyll.environment}}" markdown="1">
+{% for i in (0..9) %}
+  {% capture q %}q{{i}}{% endcapture %}
+  {% capture a %}a{{i}}{% endcapture %}
+  {% if include.[q] == nil %}{% break %}{% endif %}
+  {% assign qslug = include.[q] | slugify: 'latin' %}
+  {% if jekyll.environment == "email" %}
+   - <i markdown="1">{{include.[q]}}</i><br>{{include.[a]}}
+  {% else %}
+   - <details id="{{qslug}}" markdown="1"><summary><span markdown="1">{{include.[q]}}</span></summary>
+     {{include.[a]}}&nbsp;<a href="#{{qslug}}">🔗</a>
```

Comment:
I suggest we drop the link icon here:

- I think the most intuitive interpretation is that clicking on this will link to the discussion or source, not this section of the newsletter
- it's unlikely we (or anyone else) will need to link to these often. If we do, we can just inspect the html and grab the id from there.